ui/desktop/electron-app/config/forge.config.js

```diff
@@ -103,6 +103,9 @@ module.exports = {
           fs.mkdirSync(destination, { recursive: true });
         // Copy artifacts
         artifacts.forEach(async (artifact) => {
+          if (arch === 'x64') {
```

Comment:
I add this arch "rewrite" at the `postMake` hook, so shouldn't affect any other part of the build process.